es-elasticity.gemspec

```diff
@@ -26,16 +26,16 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "codeclimate-test-reporter"
   spec.add_development_dependency "oj"
   spec.add_development_dependency "pry"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "redis"
   spec.add_development_dependency "rspec", "~> 3.13.0"
   spec.add_development_dependency "rspec_junit_formatter"
   spec.add_development_dependency "simplecov", "~> 0.22.0"
   spec.add_development_dependency "timecop"
 
-  spec.add_dependency "activemodel",   ">= 5.2.0", "< 8"
-  spec.add_dependency "activerecord",   ">= 5.2.0", "< 8"
-  spec.add_dependency "activesupport", ">= 5.2.0", "< 8"
+  spec.add_dependency "activemodel",   ">= 5.2.0", "< 8.1"
```

Comment:
Do you think we need to be this restrictive? I don't know how all we use these 3 gems in this gem but it would be nice if we weren't blocked with each minor Rails upgrade if possible.

Reply:
Good question - I erred on the side of being more conservative, but you're right that being too restrictive can cause unnecessary upgrade blockers between minor versions. I don't see major compatibility concerns with relaxing it to `< 9` as these gems should be fairly stable across minor versions.

Reply:
Done via https://github.com/doximity/es-elasticity/pull/97/commits/8268e1d85132a01d550b7fb2e97db717d1466444